timesketch/ui/app.module.ts

```diff
@@ -25,21 +25,32 @@ import 'zone.js'
 import angular from 'angularjs-for-webpack'
 import {NgModule} from '@angular/core'
 import {BrowserModule} from '@angular/platform-browser'
+import {FormsModule} from '@angular/forms'
+import {HttpClientModule, HTTP_INTERCEPTORS} from '@angular/common/http'
 
-import 'css/ts.css'
+// add all operators from rxjs, don't care about overhead
+import 'rxjs/Rx'
+
+// cytoscape needs that to bind all events
+import 'hammerjs'
+
+import './css/ts.css'
```

Comment:
We use webpack to create a js bundle and a css bundle. The `timesketch/ui` directory is not visible from the web at all, only the `timesketch/static`. Later on, it would be easy to have multiple css files bundled by webpack - just import them all - and also, we can add `scss` transpilation easily because webpack is already set up.